db/cached.js

```diff
@@ -14,6 +14,9 @@ module.exports = function (storage) {
     find: function (query, offset, limit, orderby, fields) {
       return cache.find(query, offset, limit, orderby, fields)
     },
+    count: async function(...params) {
+      return (await this.find(...params)).length
```

Comment:
Instead this should be `return (await cache.count(...params))` since all the storage types support count :)

Reply:
thanks! fixed